clang/lib/CodeGen/ConstantInitBuilder.cpp

```diff
@@ -37,8 +37,8 @@ void ConstantInitFuture::abandon() {
 
 void ConstantInitFuture::installInGlobal(llvm::GlobalVariable *GV) {
   assert(Data && "installing null future");
-  if (Data.is<llvm::Constant*>()) {
-    GV->setInitializer(Data.get<llvm::Constant*>());
+  if (auto *C = dyn_cast<llvm::Constant *>(Data)) {
+    GV->setInitializer(C);
   } else {
     auto &builder = *Data.get<ConstantInitBuilderBase*>();
```

Comment:
Leftover get here?

Reply:
Thank you for pointing this out!  Fixed.